src/geopackage/geometry.cpp

```diff
@@ -75,7 +75,8 @@ geojson::geometry geopackage::build_geometry(
     }
 
     if (!is_empty) {
-        const std::vector<uint8_t> geometry_data(geometry_blob.begin() + index, geometry_blob.end());
+        const boost::span<const uint8_t> geometry_data(geometry_blob.data() + index,
+                                                       geometry_blob.data() + geometry_blob.size());
```

Comment:
Note to readers: `span` cannot be constructed from opaque iterators, only (begin pointer, end pointer) or (begin pointer, size)

Reply:
(or from other spans of convertible types, or from whole containers or ranges or types that it knows how to convert)

Reply:
The latter argument could also have been written as 
```suggestion
                                                       geometry_blob.size() - index);
```
But that requires more thought to understand, than just specifying the end address to mean 'everything from index onward'